apps/foundation-demo/src/app/checkout/components/shipping-async-wrapper/shipping-async-wrapper.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="shipping-async-wrapper" shipping-container #ShippingContainer="ShippingContainer">
+  <shipping-form 
+    [hidden]="!showShippingForm"
+    [shippingInfo]="ShippingContainer.shippingInfo$ | async" 
+    (updateShippingInfo)="ShippingContainer.updateShippingInfo($event); toggleShippingView()"></shipping-form>
```

Comment:
We need to remove toggleShippingView(). Please see the component.ts for further info.

Reply:
done